readme.md

```diff
@@ -456,10 +456,10 @@ import createContext from 'zustand/context'
 
 const { Provider, useStore } = createContext()
 
-const store = create(...)
+const store = () => create(...)
```

Comment:
```suggestion
const createStore = () => create(...)
```
Maybe this seems better?

Reply:
I would prefer the snippets to be in the repo. What if we added wiki snippets reference in the README.md and that person edit it later. That'll become a problem. What do you mean by <details> in README.md? any ref?